rocky/rocky/views/finding_list.py

```diff
@@ -44,3 +44,19 @@ def build_breadcrumbs(self):
                 "text": _("Findings"),
             }
         ]
+
+
+class Top10FindingListView(FindingListView):
+    template_name = "findings/finding_list.html"
+    ooi_types = {Finding}
+
+    def get_queryset(self):
+        return super().get_queryset()[:10]
```

Comment:
Do we want to hard-code 10 or make it configurable (already)?

Reply:
This is temporary, until we have a real "dashboard" design. So I would say it doesn't have to be configurable.